app/src/main/java/com/amaze/filemanager/utils/ServiceWatcherUtil.java

```diff
@@ -25,26 +30,27 @@
     private Handler handler;
     private static HandlerThread handlerThread;
     private ProgressHandler progressHandler;
-    long totalSize;
     private Runnable runnable;
 
+    public static int STATE = -1;
```

Comment:
Naming conventions, this variable in not final.

Reply:
It is not final, but refers to final values and is used with another class too. So it's easy to differentiate from a public method.

Reply:
>  but refers to final values

A lot of variable only have specific states (this is Android so instead of enums, ints are used, as a general example), that doesn't mean they are constant.

> So it's easy to differentiate from a public method.

No one will mistake a public variable for a method :/